models/reports/config/generic/sensitive-lab-requests-line-list.json

```
@@ -0,0 +1,66 @@
+  "name": "Sensitive lab requests line list",
+  "notes": "Sensitive lab requests line list\nThis report generates a list of all patients that have had a lab request containing sensitive tests created and details of the request for the selected parameters. Patients are listed in chronological order based on the lab request date and time.  \n\nThe report includes lab requests for all statuses and if a request has been cancelled will display the reason for cancellation. ",
+  "status": "published",
+  "dbSchema": "reporting",
+  "queryOptions": {
+    "defaultDateRange": "7days",
+    "dataSources": ["thisFacility", "allFacilities"],
+    "parameters": [
+      {
+        "parameterField": "PractitionerField",
+        "label": "Requesting clinician",
+        "name": "requestedById"
+      },
+      {
+        "parameterField": "LabTestCategoryField",
```

Update to new category field.
```suggestion
        "parameterField": "SensitiveLabTestCategoryField",
```